### ISSUE TITLE
[Bug] Buttons don't have useful names

### ISSUE THREAD

**Issue by Bartek532** [taniarascia/takenote#510]
**Expected behavior**
Buttons, which have inside svg graphic should have something for screen readers to read them

**Actual behavior**
Actually, screen readers announce these buttons as 'button', so users don't know how to use them

**Notes**
Some examples of bugged buttons:
![image](https://user-images.githubusercontent.com/57185551/120080520-a4206000-c0b9-11eb-9e22-7a31cea37b86.png)
